### PR TITLE
[data-explorer] Split up index into individual files in components folder

### PR DESCRIPTION
**Pull request by BenRussert**
I saw a discussion about splitting `index.js` up into a components folder so I thought I'd try it out as I read through the package. I think it's easier to follow this way, but would be interested in what you all think.

### CODE REVIEW

**Comment by rgbkrk** [comment]
If you run `yarn lint --fix` it'll address the couple of lint errors.

**Comment by lgeiger** [comment]
> If you run yarn lint --fix it'll address the couple of lint errors.

👍 Probably we should add this to our pre-commit hook too.

**Comment by BenRussert** [comment]
Woops, I'll fix those errors later today.

File: packages/transform-dataresource/src/components/Toolbar.js [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+  BoxplotIcon,
+  ScatterplotIcon,
+  LineChartIcon,
+  BarChartIcon,
+  HexbinIcon,
+  ParallelCoordinatesIcon
+} from "../icons";
+import type View from "../index.js";
+
+import { IconButton } from "./IconButton";
```
Comment (BenRussert) [inline packages/transform-dataresource/src/components/Toolbar.js:19]: I amended and force pushed to fix the eslint error. The only change was moving the type import above the `IconButton` import and adding an empty line between them.

**Review by emeeks — APPROVED** [review]

**Comment by emeeks** [comment]
Thanks for handling this, Ben!

**Pull request merged**